src/components/Upgrade/HighlightScreen.js

```diff
@@ -1,112 +1,206 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import PropTypes from 'prop-types'
-import { Button } from '@aragon/ui'
+import { Button, springs } from '@aragon/ui'
+import { Transition, animated } from 'react-spring'
+import RemoteImage from '../RemoteImage'
 
 // Ratios based on the the design files
 export const RATIO_LEFT = 500 / 1055
 export const RATIO_TOP = 560 / 950
 
+const TRANSLATE_VALUE_TITLE = 20
+const TRANSLATE_VALUE_HEADING = 40
+const TRANSLATE_VALUE_CONTENT = 60
+
+// Helping the styled-components `css` preprocessor
```

Comment:
Does this help at `animate` time? Should we do this everywhere (if using `css` within a `react-spring`)?

Reply:
It’s only for the `css` preprocessor: because `animated.xyz` components are not capitalized, it thinks they are HTML tags and transform them into into `styled("animated.xyz")` instead of `styled(animated.xyz)`… it’s probably pretty easy to fix, I don’t think there is any valid reason to do that.

So ideally it should be fixed in `styled-components`, but in the meantime, maybe we can keep doing this? We could also use a shared file to re-export all the `animated.xyz` into `AnimXyz`, I don’t really mind either way.

Reply:
❤️ for that PR!!